src/platforms/web/security.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+import Vue from 'core/index'
+import {getTrustedTypes, isTrustedValue} from 'shared/util'
+
+type TrustedTypePolicy = {
+  // value returned is actually an object with toString method returning the wrapped value
+  createHTML: (value: any) => string;
+};
+
+let policy: ?TrustedTypePolicy
+// we need this function to clear the policy in tests
```

Comment:
Is this strictly for testing only and there would be no legit use case for it by end users? In that case we can make it exposed in dev mode only by wrapping it with `if (process.env.NODE_ENV !== 'production')`